app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    existing_file = command.split(" ")[1]
```

Comment:
You can use list unpacking `copy_comand, existing_file, new_file = command.split()`. And then you can use `copy_command` for checking.